packages/account-v2/src/containers/DocumentSelection/index.tsx

```diff
@@ -11,7 +10,7 @@ type TDocumentSelection = {
 };
 
 export const DocumentSelection = ({ countryCode, handleOnClick }: TDocumentSelection) => {
-    const { isMobile } = useBreakpoint();
+    const { isMobile } = useDevice();
```

Comment:
let's not use `isMobile` coz it's still not correct from the deriv-ui, 
Invert it to `isDesktop` as with the current breakpoints you'll need to define the styles again for tablet if you use `isMobile`

Reply:
We will fix the breakpoint iteration structure once we remove `quill` completely

Reply:
I prefer using isMobile over isDesktop, coz on resizing screen smaller than min width of isDesktop, the style jumps to mobile mode. So better use isMobile check and use isDesktop as anything other than not mobile screen.